.github/workflows/cd.yml

```
@@ -36,4 +36,7 @@ jobs:
 
       - name: Build and push Docker image
         run: |
-          gcloud builds submit --tag us-central1-docker.pkg.dev/notely-471207/notely-ar-repo/notely:${{ github.sha }} .
\ No newline at end of file
+          gcloud builds submit --tag us-central1-docker.pkg.dev/notely-471207/notely-ar-repo/notely:${{ github.sha }} .
+
+      - name: Deploy to Cloud Run
+        run: gcloud run deploy notely --image us-central1-docker.pkg.dev/notely-471207/notely-ar-repo:latest --region us-central1 --allow-unauthenticated --project notely-471207 --max-instances=4
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)